pyproject.toml

```diff
@@ -135,6 +135,7 @@ xfail_strict = true
 [tool.ruff]
 fix = true
 force-exclude = true
+per-file-target-version = {"examples/*" = "py313"}
```

Comment:
Found this new setting in the blog https://astral.sh/blog/ruff-v0.12.0#version-related-syntax-errors